### PR TITLE
Add Winds 0.3.0

### PR DESCRIPTION
**Pull request by astrotars**
*If there’s a checkbox you can’t complete for any reason, that's okay, just explain in detail why you weren’t able to do so.*

After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked that the cask was not already refused in [closed issues].

[token reference]: https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/caskroom/homebrew-cask/pulls
[closed issues]: https://github.com/caskroom/homebrew-cask/issues?q=is%3Aissue+is%3Aclosed

### CODE REVIEW

File: Casks/winds.rb [hunk 1]
```diff
@@ -0,0 +1,11 @@
+cask 'winds' do
+  version '0.3.0'
+  sha256 'f3d97376a9f060c6d4c1b3f32383332cfa4a96c294aed760306111759adc292b'
+
+  # github.com was verified as official when first introduced to the cask
+  url 'https://github.com/GetStream/Winds/releases/download/v0.3.0/Winds.zip'
```
Comment (miccal) [inline Casks/winds.rb]: Can you please add an [appcast](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md)?

Comment (miccal) [inline Casks/winds.rb]: Please change this to `url "https://github.com/GetStream/Winds/releases/download/v#{version}/Winds.zip"`.

**Comment by nparsons08** [comment]
@miccal Updated :)

**Comment by miccal** [comment]
Thank you.

**Review by miccal — APPROVED** [review]

**Comment by miccal** [comment]
Thank you for your contribution.

**Pull request merged**